<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ComPortSettingsWgt</class>
 <widget class="QWidget" name="ComPortSettingsWgt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>253</width>
    <height>291</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>240</width>
    <height>100</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Настройка порта:</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="comPort_label">
        <property name="text">
         <string>Выбор порта:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comPort_comboBox"/>
      </item>
      <item>
       <widget class="QLabel" name="baudRate_label">
        <property name="text">
         <string>Бодрейт:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="baudRate_comboBox"/>
      </item>
      <item>
       <widget class="QFrame" name="frame">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="saveSettings_pushButton">
           <property name="text">
            <string>Применить</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Статус:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="current_state_label">
        <property name="text">
         <string>выключено</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
